### PR TITLE
Fixed the arrow_right.svg anf arrow_left.svg path in mdb.css

### PR DESCRIPTION
**Pull request by Valindo**
Modified `background-image` for both `.carousel .carousel-control-prev-icon` & `.carousel .carousel-control-next-icon` from `url(../img//svg/arrow_right.svg)` to `url(../img/svg/arrow_right.svg)`

While using the carousel for my website, I noticed the arrows worked locally but when uploaded to a server, they don't.
I went back to the docs to see if I missed something and found out that the docs also had the same bug ( screenshot attached )
screen shot 2018-02-25 at 2 20 27 pm 1

I hope this counts as a meaningful contribution.

Cheers all the contributors, good work :D

### CODE REVIEW

File: css/mdb.css [hunk 1]
```diff
@@ -5966,10 +5966,10 @@ body.scrollable {
   height: 2.25rem; }
 
 .carousel .carousel-control-prev-icon {
-  background-image: url(../img//svg/arrow_left.svg); }
+  background-image: url(../img/svg/arrow_left.svg); }
```
Comment (GeekJosh) [inline css/mdb.css]: `background-image: url(#{$image-path}/svg/arrow_left.svg); }`

File: css/mdb.css [hunk 2]
```diff
@@ -5966,10 +5966,10 @@ body.scrollable {
...
 
 .carousel .carousel-control-next-icon {
-  background-image: url(../img//svg/arrow_right.svg); }
+  background-image: url(../img/svg/arrow_right.svg); }
```
Comment (GeekJosh) [inline css/mdb.css]: `background-image: url(#{$image-path}/svg/arrow_right.svg); }`

**Comment by mannok** [comment]
How come this is not yet fixed?

**Comment by mattonit** [comment]
This issue was already fixed in #a237fb3

**Pull request closed without merging**